styles.css

```diff
@@ -0,0 +1,108 @@
+@import url('https://memorable-ui.netlify.app/styles/main.css');
+
+
+body {
+    background-color: var(--primary-color);
+}
+
+p {
+    font-size: 1.6rem;
+    line-height: 1.5;
+}
+
+img {
+    width: 100%;
+}
+
+.container {
+    max-width: 900px;
```

Comment:
be consistent with the use of units you have used 'px' and 'rem'.

Reply:
Done